src/platforms/javascript/guides/react/features/error-boundary.mdx

```diff
@@ -65,6 +67,12 @@ By default React [logs all errors to the console](https://github.com/facebook/re
 
 </Alert>
 
+## Linked Errors
```

Comment:
l: iirc there's something special about the parent/child relationship here. Should we mention this?

Reply:
Found what I had in mind: https://github.com/getsentry/sentry-javascript/issues/8871

Reply:
yeah good point I'll write that down.